playwright/jest-setup.js

```diff
@@ -0,0 +1,14 @@
+const { getTestRunnerConfig, setPreRender, setPostRender } = require('../dist/cjs');
+
+const testRunnerConfig = getTestRunnerConfig(process.env.STORYBOOK_CONFIG_DIR);
+if (testRunnerConfig) {
+  if (testRunnerConfig.setup) {
+    testRunnerConfig.setup();
+  }
+  if (testRunnerConfig.preRender) {
+    setPreRender(testRunnerConfig.preRender);
```

Comment:
Do you think it makes sense to add logging to this so people can assert whether their configuration was working correctly? Maybe shown in certain log level (I guess something for the future)

Reply:
makes sense